src/main/java/sirius/biz/tenants/UserAccount.java

```diff
@@ -44,32 +44,6 @@
      */
     UserAccountData getUserAccountData();
 
-    /**
```

Comment:
This interface doesn't extend Entity yet

Reply:
Or maybe RateLimitedEntity should extend Entity instead?

Reply:
Kommt von TenantAware